### DISCUSSION TITLE
Charges for Multiple Db/Projects in Pro account

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#28042] [Questions] [answered]
Hi,

Can I have a Production project/DB and a UAT project/DB in the same pro plan and will there be one charge of $25 or two sets of $25 (making $50) or does it depend on combined ingress?

**Comment by GaryAustin1** [comment, marked as answer]
You pay additional compute time only (with shared egress/storage limits/costs).  So if you have two of the smallest instances then your cost would be $25 for the first instance and $10 for compute for the 2nd instance in the org.
